### ISSUE TITLE
Add ubuntu binaries to release

### ISSUE THREAD

**Issue by G2Pavon** [seedee/SDHLT#15]
Currently you can get the Ubuntu binaries through the artifacts of completed workflows in Actions, but is not an easy way to access as it requires a github account

**Comment by Stalker2106** [comment]
Agreed, I just had to build the whole thing myself, because I haven't thought about checking actions artifacts
